app/AppCache.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
+{
+
+$ddddddddddddddddddddddddddddddddddddddddddddddddddd= '';
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected at least 4 spaces, found 0
* Visibility must be declared on property "$ddddddddddddddddddddddddddddddddddddddddddddddddddd"
* Scope modifier not specified for member variable "$ddddddddddddddddddddddddddddddddddddddddddddddddddd"
* Add a single space around assignment operators